spec/helpers/hyrax/work_form_helper_spec.rb

```diff
@@ -32,6 +32,40 @@
     end
   end
 
+  describe '.form_tab_label_for' do
+    context 'with a change set style form' do
+      let(:work) { build(:hyrax_work) }
```

Comment:
is all this stubbing necessary?  since the form isn't used in the default implementation, could we  do `let(:form) { double('form') }` and drop the rest?